recipes/Text_to_Shell/Text_to_Shell.ipynb

```diff
@@ -91,31 +214,79 @@
     "Question:\n",
     "create a tar ball of all pdf files in current folder\n",
     "Answer:\n",
-    "find . -name *.pdf | xargs tar czvf /root/Desktop/evidence/pdf.tar\n",
+    "find . -name *.pdf | xargs tar czvf pdf.tar\n",
     "\n",
     "Question:\n",
-    "Sort all files/directories under current directory according to modification time and print only the recent 7 of them\n",
+    "Sort all files/directories in the current directory, but no subdirectories, according to modification time and print only the recent 7 of them\n",
     "Answer:\n",
-    "find -mindepth 1 -printf \\\"%T@ %Pn\\\" | sort -n -r | cut -d' ' -f 2- | tail -n +7\n",
+    "find . -maxdepth 1 -exec stat -f \"%Sm {}\" \\; | sort -n -r | tail -n 7\n",
     "\n",
     "Question:\n",
     "find all the empty directories in the current folder\n",
     "Answer:\n",
     "find . -type d -empty\n",
     "\n",
-    "\"\"\"\n",
-    "\n",
-    "query = \"How can I use a bash script to calculate the number of days between the dates Jan 12, 2023 and Jan 20, 2024?\"\n",
```

Comment:
Makes sense to remove this. I think the examples and the prompts should be directives (eg., "Write some code") vs questions ("How do I?").